marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.test.tsx

```diff
@@ -96,6 +96,7 @@ describe("Mapping Card component", () => {
     expect(getByRole('delete-mapping')).toBeInTheDocument();
     await fireEvent.click(getByRole('delete-mapping'));
     await fireEvent.click(getByText('Yes'));
+    expect(getByText(/Are you sure you want to delete the /i)).toBeInTheDocument();
```

Comment:
This is an incomplete test, since the change was to verify name of the step in the confirmation modal. Please search and check usage of getSubElements and use it in the test to verify complete text with bold tags in them.